### PR TITLE
Pagination

### PR DESCRIPTION
**Pull request by dcarrilloWolox**
## Summary ##
* I installed wor-pagination gem
## Trello Card ##
https://trello.com/c/Z7ScsqVM/34-agregar-paginaci%C3%B3n-a-index-de-book-y-tests

### CODE REVIEW

File: app/controllers/api/v1/users_controller.rb [hunk 1]
```diff
@@ -1,9 +1,12 @@
 module Api
   module V1
     class UsersController < ApiController
+      include Wor::Paginate
+      before_action :current_user
```
Comment (Martum) [inline app/controllers/api/v1/users_controller.rb]: Why are you doing this??

File: app/controllers/api_controller.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@ class ApiController < ApplicationController
   rescue_from ActionController::ParameterMissing, with: :render_nothing_bad_req
   rescue_from ActiveRecord::RecordNotFound, with: :render_nothing_bad_req
   protect_from_forgery with: :null_session
-  before_action :current_user, :authenticate_request, :set_locale
+  before_action :current_user, :authenticate_request
```
Comment (Martum) [inline app/controllers/api_controller.rb]: Why did you remove `:set_locale`?

File: spec/controllers/api/v1/users_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+describe Api::V1::UsersController do
```
Comment (Martum) [inline spec/controllers/api/v1/users_controller_spec.rb:3]: Its fine, but also use the pagination [helpers](https://github.com/Wolox/wor-paginate#test-helpers) included in the gem.

**Review by Martum — CHANGES_REQUESTED** [review]

File: app/controllers/api_controller.rb (hunk 2 above)
Comment (Martum) [reply app/controllers/api_controller.rb]: `I18n.locale = current_user&.locale || I18n.default_locale`

`!current_user.nil? && current_user.locale`

File: app/controllers/api/v1/users_controller.rb (hunk 1 above)
Comment (dcarrilloWolox) [reply app/controllers/api/v1/users_controller.rb]: my bad it was :set_locale, but i'll move this to apiController

**Review by Martum — APPROVED** [review]

**Pull request merged**